pkg/stacks/unpack_test.go

```diff
@@ -1176,7 +1243,10 @@ func TestUnpack(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			got := &bytes.Buffer{}
 			rd := &walker.ResourceDir{Base: tt.root, Walker: afero.Afero{Fs: tt.fs}}
-			err := Unpack(rd, got, tt.root, "Namespaced", "crossplane/ts-controller:0.0.0")
+
+			os.Setenv(StackImageEnv, "crossplane/sample-stack:0.0.1")
```

Comment:
I think the tests would benefit from `StackImageEnv`'s value being set in the `tests` struct. We can toggle different values for each row in the the same test table.  As a string, the default value (for the existing tests that don't specify `tt.stackImageEnv`) will be "", which is the same as being unset.

Reply:
Ah, yeah, that's a better approach

Reply:
This is done; resolving